modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -274,13 +274,8 @@ Use the following commands to manage applications deployed to Runtime Fabric.
 |===
```

Comment:
Should also remove line 9 and 10:
* Perform a heap dump or a thread dump of a running application.
* Display memory usage of a running application.